lib/azure/resource_management/ARM_interface.rb

```diff
@@ -29,17 +29,17 @@ class ARMInterface < AzureInterface
       include Azure::ARM::ARMDeploymentTemplate
       include Azure::ARM::VnetConfig
 
-      include Azure::ARM::Resources
-      include Azure::ARM::Resources::Models
+      include Azure::Resources::Mgmt::V2018_05_01
+      include Azure::Resources::Mgmt::V2018_05_01::Models

```

Comment:
https://github.com/Azure/azure-sdk-for-ruby/commit/096d78b680d1d5ac1986947ce16aeef122273686

https://rubygems.org/gems/azure_mgmt_resources/versions/0.17.0